micromanager_gui/multid_widget.py

```diff
@@ -117,6 +153,45 @@ def _set_enabled(self, enabled: bool):
         self.stage_pos_groupBox.setEnabled(enabled)
         self.acquisition_order_comboBox.setEnabled(enabled)
 
+    def _set_top(self):
+        self.z_top_doubleSpinBox.setValue(self._mmc.getZPosition())
+
+    def _set_bottom(self):
+        self.z_bottom_doubleSpinBox.setValue(self._mmc.getZPosition())
+
+    def _update_topbottom_range(self):
+        self.z_range_topbottom_doubleSpinBox.setValue(
+            abs(self.z_top_doubleSpinBox.value() - self.z_bottom_doubleSpinBox.value())
+        )
+
+    def _update_rangearound_label(self):
+        up_down_range = self.zrange_spinBox.value() / 2
+        self.range_around_label.setText(
+            f"-{up_down_range} µm <- z -> +{up_down_range} µm"
+        )
+
+    def _update_abovebelow_range(self):
+        self.z_range_abovebelow_doubleSpinBox.setValue(
+            self.above_doubleSpinBox.value() + self.below_doubleSpinBox.value()
+        )
+
+    def _update_n_images(self, value):
+        # when chenging step_size_doubleSpinBox
+        if self.sender().objectName() == "step_size_doubleSpinBox":
+            step = value
+        else:
+            step = self.step_size_doubleSpinBox.value()
```

Comment:
can this just be:  `step = self.step_size_doubleSpinBox.value()`?  is that the intention here?